pulp_rpm/app/migrations/0039_disttree_digest.py

```diff
@@ -1,8 +1,25 @@
 # Generated by Django 3.2.9 on 2021-12-20 22:05
 
+import random
+
 from django.db import migrations, models
 
 
+def generate_fake_digest(apps, schema_editor):
+    """
+    Generate a temporary digest for a dist tree
+
+    This is needed so the dist tree is fixed by a subsequent sync.
+    We can't use digest from .treeinfo file because a dist tree object is potentially wrong and
+    we need to let a subsequent [force] sync to fix it.
+    """
+    DistributionTree = apps.get_model('rpm', 'DistributionTree')
+    dist_trees = DistributionTree.objects.all()
+    for dist_tree in dist_trees:
+        dist_tree.digest='random %032x, to be removed at sync' % random.randrange(16**32)
```

Comment:
If this just needs to be a unique string, i would rather use a serial number than random. (It may be faster, but most importantly, it's guaranteed to not produce collisions.)

Reply:
@goosemania That seems reasonable to me ^